manifests/server/config_ini.pp

```diff
@@ -0,0 +1,51 @@
+# PRIVATE CLASS - do not use directly
+class puppetdb::server::config_ini (
+  $command_threads = $puppetdb::params::command_threads,
+  $store_usage     = $puppetdb::params::store_usage,
+  $temp_usage      = $puppetdb::params::temp_usage,
+  $confdir         = $puppetdb::params::confdir,
+) inherits puppetdb::params {
+
+  # Set the defaults
+  Ini_setting {
+    path    => "${confdir}/config.ini",
+    ensure  => present,
+    section => 'command-processing',
+  }
```

Comment:
You may need to manage the file resource itself as well, since a PDB package might not have it there by default.